CMakeLists.txt

```diff
@@ -338,6 +338,21 @@ else()
   else()
     usFunctionCheckCompilerFlags("-fvisibility=hidden -fvisibility-inlines-hidden" _have_visibility)
   endif()
+  
+  # Issue #151: Since we explicitly compile CMakeResourceDependencies.cpp, we cache the OSX specific compiler flags
+  if(APPLE)
+    set(US_OSX_CXX_FLAGS ) 
+    if(CMAKE_OSX_SYSROOT)
+      list(APPEND US_OSX_CXX_FLAGS -isysroot ${CMAKE_OSX_SYSROOT})
+    endif()
+    if(CMAKE_OSX_ARCHITECTURES)
+      list(APPEND US_OSX_CXX_FLAGS -arch ${CMAKE_OSX_ARCHITECTURES})
+    endif()
+    if(CMAKE_OSX_SYSROOT)
```

Comment:
This looks like a copy and paste error, should be `if(CMAKE_OSX_DEPLOYMENT_TARGET)`